app/assets/stylesheets/layouts/starrylight.scss

```diff
@@ -73,7 +73,7 @@ $bg_color_linkbox_subber: $colors_wildblueyonder;
 
 // other Buttons
 $bg_color_viewbutton: $colors_scampi;
-$bg_color_viewbutton_selected: $colors_scampi;
+$bg_color_viewbutton_selected: $colors_wildblueyonder;
```

Comment:
What's with this color change?

Reply:
"Incidentally fixes a...lack of distinct coloring for .view-button.selected in Starry Light layouts" < this bit. In the process of testing my various new character link options for the character_menu_link I noticed that, because this color was originally the same as the one above it, the .selected class wasn't actually doing anything visible at all. It is now visually distinct!

Reply:
Aha, yeah, I missed that. Cool.